Dynavity/Dynavity/view-model/canvas/CanvasViewModel.swift

```diff
@@ -223,3 +237,42 @@ extension CanvasViewModel {
         }
     }
 }
+
+// MARK: Annotation palette controls button handlers
+extension CanvasViewModel {
+    func selectNoAnnotationTool() {
```

Comment:
I think this function could be renamed to something like `clearSelectedAnnotationTool`?